students/AndrewMiotke/lesson02/assignment/src/charges_calc.py

```diff
@@ -0,0 +1,95 @@
+'''
+Returns total price paid for individual rentals
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+from logging.config import dictConfig
+
+"""
+NOTES FROM CLASS - DELETE THESE LATER
+Create 2 file handlers
+Create a if/elif statement for each logging level
+"""
+
+# Logging layout
+LOG_FORMAT = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
+FORMATTER = logging.Formatter(LOG_FORMAT)
+FILE_HANDLER = logging.FileHandler('charges_calc.log')
+
+# Error
+ERROR_LOGGER = FILE_HANDLER
+ERROR_LOGGER = logging.getLogger()
+ERROR_LOGGER.addHandler(FILE_HANDLER)
+
+# Warning
+WARNING_LOGGER = FILE_HANDLER
+WARNING_LOGGER = logging.getLogger()
+WARNING_LOGGER.addHandler(FILE_HANDLER)
+
+# DEBUG
+DEBUG_LOGGER = FILE_HANDLER
+DEBUG_LOGGER = logging.getLogger()
+DEBUG_LOGGER.addHandler(FILE_HANDLER)
+FILE_HANDLER.setLevel(logging.DEBUG)
+FILE_HANDLER.setFormatter(FORMATTER)
+
+
+def parse_cmd_arguments():
+    """ Command line arguements to start the program """
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument('-o', '--output', help='ouput JSON file', required=True)
+    parser.add_argument('-d1', '--debug1', action='store_true', help='debug error messages')
```

Comment:
Instead of having three different arguments here, it will be easier to just have 1 argument which will accept 0-3 as possible values.